doc/particles.md

```diff
@@ -0,0 +1,393 @@
+# Particles
+
+Flame offerst a basic, yet robust and extendable particle system. The core concept of this system is `Particle` class, which is very similar in its behavior to the `Component`.
```

Comment:
small typo: `offers`

add a `the` before the Particle on this sentence

> The core concept of this system is `Particle`


Here: 
> which is very similar in its behavior to the `Component`.

My suggestion:

which is very similar in its behaviour to the `ParticleComponent`.

Reply:
Thanks, fixed!